corehq/apps/reports/standard/monitoring.py

```diff
@@ -515,7 +515,7 @@ def rows(self):
 
     @property
     def get_all_rows(self):
-        es_results = self.es_queryset(user_ids=self.owner_ids)
+        es_results = self.es_queryset(owner_ids=self.owner_ids)
```

Comment:
was this a bug?

Reply:
It was. Silly mistake on my end where I forgot to update the param name for the function call.

Reply:
Cool, no worries. Glad that it was caught :+1: